lib/xsound/ogg_stream.cpp

```diff
@@ -37,8 +37,8 @@ static long   stream_tell_func  (void *datasource) {
 	return ftell(file);
 }
 
-OggStream::OggStream(const std::string &fname) {
-	_file = fopen(fname.c_str(), "rb");
+OggStream::OggStream(const std::string_view fname) {
+	_file = fopen(fname.data(), "rb");
```

Comment:
why data is more preferable then c_srr()?

Reply:
simply because there is no such things with string_view :-) it s chunk of bytes that s how it gains perf over a traditional std string, no need allocation.